src/main/java/org/minions/devfund/bruno/moviedatabase/MovieDatabase.java

```diff
@@ -0,0 +1,239 @@
+package org.minions.devfund.bruno.moviedatabase;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+/**
+ * Class manages the data base movie.
+ */
+public class MovieDatabase {
+    private static final String DATABASE_MOVIES_PATH = "src/main/resources/bruno/movies.txt";
+    private static final String DATABASE_RATING_PATH = "src/main/resources/bruno/rating.txt";
+    private List<Actor> actorList;
+    private List<Movie> movieList;
+
+    /**
+     * Constructor.
+     */
+    public MovieDatabase() {
+        actorList = new LinkedList<>();
+        movieList = new LinkedList<>();
+    }
+
+    /**
+     * Adds movie to the data base.
+     *
+     * @param movieName String.
+     * @param actors    Array.
+     */
+    public void addMovie(final String movieName, final String[] actors) {
+        List<String> actorsArray = Arrays.asList(actors);
+        actorsArray.forEach(actorName -> addActor(actorName, movieName));
+    }
+
+    /**
+     * Adds the actor according if the actor name was added before.
+     *
+     * @param actorName String to set in Movie object.
+     * @param movieName String.
+     */
+    private void addActor(final String actorName, final String movieName) {
+        Actor actor = getAnActor(actorName);
+        if (actor != null) {
+            generatesTheMovies(actor, movieName);
+        } else {
+            Actor newActor = new Actor();
+            newActor.setName(actorName);
+            generatesTheMovies(newActor, movieName);
+            actorList.add(newActor);
+        }
+    }
+
+    /**
+     * Adds Rating to a movie.
+     *
+     * @param name   String movie.
+     * @param rating Double rating.
+     */
+    public void addRating(final String name, final double rating) {
+        Movie movie = getAMovie(name);
+        if (movie != null) {
+            movie.setRating(rating);
+        }
+    }
+
+    /**
+     * Updates the movie rating.
+     *
+     * @param name      String movie.
+     * @param newRating Double new Rating.
+     */
+    public void updateRating(final String name, final double newRating) {
+        Objects.requireNonNull(movieList.stream().filter(movie -> movie.getName().equals(name)).findFirst()
+                .orElse(null)).setRating(newRating);
```

Comment:
Don't return null objects
because you will need to handle the null object or probably you will cause null pointer exceptions.
and that is wrong